radio/src/targets/taranis/hal.h

```diff
@@ -890,6 +935,15 @@
   #define HARDWARE_SWITCH_F
   #define SWITCHES_GPIO_REG_F           GPIOB
   #define SWITCHES_GPIO_PIN_F           LL_GPIO_PIN_5 // PB.05
+#elif defined(RADIO_MT12)
+  // ADC based switch/pot
+#elif defined(RADIO_TPRO)
+  // SW2
+  #define FUNCTION_SWITCH_2             SF
+  #define STORAGE_SWITCH_F
+  #define HARDWARE_SWITCH_F
+  #define SWITCHES_GPIO_REG_F           GPIOG
+  #define SWITCHES_GPIO_PIN_F           LL_GPIO_PIN_6  // PG.06
```

Comment:
Was this something that got missed before with the TPRO somehow? I only ask as it seems out of place :shrug: